json-rpc/types/src/views.rs

```diff
@@ -1011,8 +1008,5 @@ mod tests {
                 }
             })
         );
-
-        let preburn: jsonrpc::PreburnWithMetadata = serde_json::from_value(value).unwrap();
-        assert_eq!(preburn.metadata, "");
```

Comment:
We need this to confirm the data rendered can be loaded back by protobuf structs.
We don't have other places cover this yet as we all moved to use same view structs.

Reply:
I didn't remove this test, if you look at the proto crate added in this commit, that test is moved there